### DISCUSSION TITLE
Christoph Nakazawa

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#170] [Announcements]
## Background

- https://twitter.com/cpojer
- https://cpojer.net/
- https://athenacrisis.com/

## Intro

Andrew: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

Justin: 

## Topics

- jest
- metro
- yarn 
- mootools
- Athena Crisis

## Questions

- [x] You've work on an amazing set of OSS projects. What drove you to focus so much on OSS tooling? Was that a part of the facebook culture?
- [x] What action did you take to make those projects so successful?
- [x] You've frequently said jest could be more than a test runner. What do you mean by that? What could jest have been?
- [x] Of the tools you worked on what's your favorite? 
- [x] What modern tools are you excited about?
- [ ] ~What are some of the your favorite features you championed on one of the tools?~
- [x] The definition of a front end developer is a funny one. It isn't just product engineers. You've categorized front end into 4 different archetypes. What are they and which one do you think you are?

### Athena Crisis

- [x] What is athena crisis and why did you choose to build it?
- [x] What is the front end built in?
- [x] Were some things easier to build because you chose html/js/css?
- [x] What part of the game did you have the most fun building?
- [ ] ~How did you accomplish multiplayer?~
- [x] What tools did you build to build the game? (testing)
- [x] What open source libraries have come out of this?
- [x] You’re building a company where the first product is Athena crisis. Congrats? What are your goals with the company?
- [x] Do you have an idea about the next game you want to build?

**Comment by cpojer** [comment]
https://nakazawa.dev
https://cpojer.net/posts/building-a-company

😄

**Comment by hipstersmoothie** [comment]
- https://charm.sh/
  - https://github.com/charmbracelet/bubbletea
  - https://github.com/charmbracelet/soft-serve 
- https://github.com/XingangPan/DragGAN

**Comment by just-be-dev** [comment]
- https://evervault.com/
- https://www.reddit.com/r/functionalprint/comments/13o24p4/made_a_soldering_station_to_adapt_m18_milwaukee/

**Comment by cpojer** [comment]
Tooltips:
* [Not Without My Ghosts](https://twitter.com/cpojer/status/1660085470745997313)
* [Mamachari](https://twitter.com/cpojer/status/1659366358541549568)
